SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -309,4 +297,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/** QBI pairs for ratio of - net cull distance : update frequency */
 	UPROPERTY(EditAnywhere, Config, Category = "Interest", meta = (EditCondition = "bEnableNetCullDistanceFrequency"))
 	TArray<FDistanceFrequencyPair> InterestRangeFrequencyPairs;
+
+public:
+	// UI Hidden settings passed through from SpatialGDKEditorSettings
+	bool bUseDevelopmentAuthenticationFlow;
```

Comment:
would it be better to move the settings completely to the Editor Settings, have a check in the SpatialWorkerConnection whether we are running in-editor or not and only apply the dev auth token editor settings if we are running in the editor

Reply:
Spoke about offline